setup.py

```diff
@@ -722,11 +722,7 @@ def pxd(name):
       maintainer=AUTHOR,
       version=versioneer.get_version(),
       packages=find_packages(include=['pandas', 'pandas.*']),
-      package_data={'': ['data/*', 'templates/*'],
```

Comment:
Anyone know what the `data/*` files were referencing? I think it was supposed to be `pandas/tests/data`?

Reply:
IIRC this looks for a data and template sub directory in any of the packages. It doesn’t refer to just one directory. 

FWIW I think if those sub directories had an init they would automatically be included per the line above this, but given that’s not the case this helps include those folders relative to any of the packages that are found